pipelines/fbc-release/README.md

```diff
@@ -15,6 +15,9 @@ Tekton release pipeline to interact with FBC Pipeline
 | verify_ec_task_bundle           | The location of the bundle containing the verify-enterprise-contract task                                | No        | -                                               |
 | postCleanUp                     | Cleans up workspace after finishing executing the pipeline                                               | Yes       | true                                            |
 
+### Changes since 1.2.0
```

Comment:
Can we switch to `Changes in 1.3.0`? I vaguely remember us agreeing to use that style instead (and I see your task change does)

Reply:
done.

Reply:
FTR, we have a jira to change this everywhere: https://issues.redhat.com/browse/RHTAPREL-669

But I agree to start using it like this already.